### PR TITLE
adding blogs page

### PR DESCRIPTION
**Pull request by MAQ-AkshayaK**

### CODE REVIEW

File: blogs.html [hunk 1]
```diff
@@ -0,0 +1,569 @@
+﻿<!DOCTYPE html>
+<!-- Microdata markup added by Google Structured Data Markup Helper. -->
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml"
+xmlns:fb="http://ogp.me/ns/fb#">
+<head>
+    <meta charset="utf-8" />
+    <title>Blogs | MAQ Software </title>
+    <meta name="title" content="Top Stories and Latest News | MAQ Software " />
```
Comment (MAQHatimD) [inline blogs.html]: Update title

File: blogs.html [hunk 2]
```diff
@@ -0,0 +1,569 @@
...
+    <meta name="theme-color" content="#c00000"/>
+    <meta name="description" content="Catch up on our latest blog posts. "/>
+    <meta name="keywords" content="maq software, innovative software, data management, Power BI, Sharepoint, cloud, artificial intelligence"
```
Comment (MAQHatimD) [inline blogs.html]: update keywords

File: blogs.html [hunk 3]
```diff
@@ -0,0 +1,569 @@
...
+    <meta charset="utf-8" />
+    <title>Blogs | MAQ Software </title>
+    <meta name="title" content="Top Stories and Latest News | MAQ Software " />
+    <meta name="theme-color" content="#c00000"/>
+    <meta name="description" content="Catch up on our latest blog posts. "/>
+    <meta name="keywords" content="maq software, innovative software, data management, Power BI, Sharepoint, cloud, artificial intelligence" 
+    />
+    <meta name="ROBOTS" content="FOLLOW,INDEX" />
+    <!-- Open Graph data -->
+    <meta name="og:title" property="og:title" content="Top Stories and Latest News | MAQ Software "
```
Comment (MAQHatimD) [inline blogs.html]: update the title tag

File: css/style.css [hunk 4]
```diff
@@ -5815,4 +5815,24 @@ li.h4header {
...
+    text-overflow: ellipsis;
+    overflow: hidden;
+}
+
+.readmoreblog {
+    float:right;
+    font-weight:600;
+}
+.blog-ellipsis{
+    width: -webkit-fill-available;
```
Comment (MAQHatimD) [inline css/style.css]: Test in cross browsers

File: js/common.js [hunk 5]
```diff
@@ -123,4 +123,14 @@ function cloneNavDropDowns () {
         $(this).remove();
         menu_clone.appendTo(parent);
     });
+}
+
+function getBlogData(blogUrl, successCallBack, completeCallBack) {
```
Comment (MAQHatimD) [inline js/common.js]: rename as only url not specific to blog

File: css/style.css (hunk 4 above)
Comment (MAQ-AkshayaK) [reply css/style.css]: tested works as expected

File: blogs.html (hunk 1 above)
Comment (MAQ-AkshayaK) [reply blogs.html]: updated

**Pull request merged**